BDArmory/Radar/RadarUtils.cs

```diff
@@ -1433,6 +1431,7 @@ public static ViewScanResults GuardScanInDirection(MissileFire myWpnManager, Tra
                             }
                             else
                             {
+                                if (vesselDistance > myWpnManager.guardRange * myWpnManager.guardRange) continue;
```

Comment:
Can we rename `vesselDistance` to `vesselDistanceSqr` for readibility?

Reply:
Yes.